packages/datadog-plugin-aws-sdk/test/s3.spec.js

```diff
@@ -74,6 +74,138 @@ describe('Plugin', () => {
           rawExpectedSchema.outbound
         )
 
+        describe('span pointers', () => {
+          it('should add span pointer for putObject operation', (done) => {
+            agent.use(traces => {
+              try {
+                const span = traces[0][0]
+                const links = JSON.parse(span.meta?.['_dd.span_links'] || '[]')
+
+                expect(links).to.have.lengthOf(1)
+                expect(links[0].attributes).to.deep.equal({
+                  'ptr.kind': S3_PTR_KIND,
+                  'ptr.dir': SPAN_POINTER_DIRECTION.DOWNSTREAM,
+                  'ptr.hash': '6d1a2fe194c6579187408f827f942be3',
+                  'link.kind': 'span-pointer'
```

Comment:
nano-nit: it might make sense to pull in a constant for this particular `link-kind` check?

Reply:
Gonna hold off on that for now because any commit will remove all approvals. This is something I can add in the next PR